src/main/java/teammates/ui/pagedata/InstructorFeedbackResultsPageData.java

```diff
@@ -1781,4 +1785,24 @@ private int generateIndexForUser(String name, int index, Map<String, Integer> us
         return index + 1;
     }
 
+    /**
+     * Checks if we should show No Specific Section panel.
+     *
+     * @return
+     * <ul>
+     * <li>true if the course has teams in Default Section</li>
+     * <li>true if view is RGQ or RQG and there is General Feedback or Feedback to or from Instructors</li>
+     * <li>false otherwise</li>
+     * </ul>
+     */
+    private boolean hasNoSpecificSectionPanel(InstructorFeedbackResultsPageViewType viewType) {
```

Comment:
I feel that this method can be left here, because it's about panel. It's clearer this way.

Reply:
We can let `getSectionFromRoster` return `Const.NO_SPECIFIC_SECTION`. But, in this specific case, wouldn't `NO_SPECIFIC_RECIPIENT` be clearer?

Reply:
`NO_SPECIFIC_SECTION` can also refer to the case where students are not in any section, but in the default section.

Reply:
`NO_SPECIFIC_SECTION` is different from `NO_SPECIFIC_RECIPIENT`. For a feedback to instructor, we can say that is should belong to `NO_SPECIFIC_SECTION` because instructor should be longs to no specific section. However, we cannot say it is no specific recipient because instructor is the recipient.

Reply:
Hmm. Okay. Code wise it wouldn't make too much of a difference I think. Naming wise based on your chain of thought, it makes sense to change it to `NO_SPECIFIC_SECTION` then.

Reply:
> I feel that this method can be left here, because it's about panel. It's clearer this way.
You can name it in different way like `hasEntitiesInNoSpecificSection`